### PR TITLE
262177:Environments files

### PR DESCRIPTION
**Pull request by ppotturi**

### CODE REVIEW

**Review by ganeshg2491 — DISMISSED** [review]

**Comment by ganeshg2491** [comment]
DoneImDoneGIF

File: services/ffc-demo/ffc-demo-web/infra/snd/02/patch.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    fluxConfigNamespace: ${FLUX_CONFIG_NAMESPACE}
+    subscriptionId: ${SUBSCRIPTION_ID}
+    serviceBusResourceGroupName: ${SERVICES_INFRA_RG}
+    serviceBusNamespaceName: ${SHARED_SERVICEBUS_NAME}
+    clusterOIDCIssuerUrl: ${CLUSTER_OIDC_ISSUER_URL}
+    teamResourceGroupName: ${TEAM_RG}
+    teamMIPrefix: ${TEAM_MI_PREFIX}
+    serviceName: ${SERVICE_NAME}
+    keyVaultResourceGroupName: ${SERVICES_INFRA_RG}
+    keyVaultName: SNDADPINFVT1401
```
Comment (ganeshg2491) [inline services/ffc-demo/ffc-demo-web/infra/snd/02/patch.yaml:21]: should this be SNDADPINFVT1402?

File: services/ffc-demo/ffc-demo-calculation-service/infra/snd/03/patch.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    fluxConfigNamespace: ${FLUX_CONFIG_NAMESPACE}
+    subscriptionId: ${SUBSCRIPTION_ID}
+    serviceBusResourceGroupName: ${SERVICES_INFRA_RG}
+    serviceBusNamespaceName: ${SHARED_SERVICEBUS_NAME}
+    clusterOIDCIssuerUrl: ${CLUSTER_OIDC_ISSUER_URL}
+    teamResourceGroupName: ${TEAM_RG}
+    teamMIPrefix: ${TEAM_MI_PREFIX}
+    serviceName: ${SERVICE_NAME}
+    keyVaultResourceGroupName: ${SERVICES_INFRA_RG}
+    keyVaultName: SNDADPINFVT1401
```
Comment (ganeshg2491) [inline services/ffc-demo/ffc-demo-calculation-service/infra/snd/03/patch.yaml:21]: Should this be SNDADPINFVT1403?

**Review by ganeshg2491 — APPROVED** [review]

**Review by jeevankuduvaravindran — APPROVED** [review]
TheBiggestOneCillianMurphyGIF

**Pull request merged**